terraform/app/modules/paas/data.tf

```diff
@@ -7,11 +7,15 @@ data cloudfoundry_space space {
   org  = data.cloudfoundry_org.org.id
 }
 
+data cloudfoundry_space prod_space {
```

Comment:
Expected the Dev and/or Test spaces to be referenced for this exploratory work, and not 'prod_space'

Reply:
As I've said, the private server  **ONLY** resides in the prod space.